server/controllers/booking/booking.controller.js

```diff
@@ -43,13 +50,60 @@ const createBooking = function(req, res) {
             console.log(validateErrs);
             res.status(500).send(errs);
           } else {
-            newBooking.save((err, booking) => {
-              if (err) {
-                res.status(504).send(err);
-              } else {
-                res.json(booking._id);
-              }
-            });
+            Booking.find({ car: req.body.car, disabled: false })
+              // Placeholder, should look for booking belonging to current user
+              .exec((err, bookings) => {
+                if (err) {
+                  res.status(500).send(err);
```

Comment:
Looks like it's all OK you do not create a new booking. Is it intentional?

Reply:
No :) Great catch. I should definitely create a booking if no booking is found. I will fix that later tonight